src/lib/tasks/server/load_server_file_task.cpp

```diff
@@ -8,8 +12,21 @@ namespace opossum {
 
 void LoadServerFileTask::_on_execute() {
   try {
-    const auto table = load_table(_file_name, Chunk::MAX_SIZE);
-    StorageManager::get().add_table(_table_name, table);
+    std::vector<std::string> file_parts;
```

Comment:
From the looks of it, this will silently do nothing if the file extension is unknown. Can you make the server reply with an error if one attempts to, e.g., load `table.xls`?

Reply:
Mmmh, Excel support. Don't brush that idea off so quickly! ;)